docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+services:
+    app:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        command: >
+            sh -c 'python manage.py wait_for_db &&
+                   python manage.py makemigrations && 
+                   python manage.py migrate &&
+                   python manage.py runserver 0.0.0.0:8000'
+        env_file:
+            - .env
+        depends_on:
+            - db
+        volumes:
+            - ./:/app/
```

Comment:
The volume mapping `- ./:/app/` should be consistent with the `WORKDIR` in your Dockerfile. If you change `WORKDIR` to `/app`, ensure this mapping reflects that change.